README.md

```diff
@@ -34,56 +46,39 @@ yarn changeset version
 
-### Contributing
+### @hashicorp/flight-icons
 
-#### @hashicorp/flight-icons
+See [flight-icons/README](packages/flight-icons/README.md)  and [flight-icons/CONTRIBUTING](packages/flight-icons/CONTRIBUTING.md) guide for details on how to use the "sync/build" scripts, that export the assets from Figma and generate a bundle of standalone SVG files.
 
-See [flight-icons/CONTRIBUTING](packages/flight-icons/CONTRIBUTING.md) guide for details on how to use the "sync/build" scripts, that export the assets from Figma and generate a bundle of standalone SVG files.
+### @hashicorp/design-system-tokens
 
-#### @hashicorp/ember-flight-icons
+See [tokens/README](packages/tokens/README.md)  and [tokens/CONTRIBUTING](packages/tokens/CONTRIBUTING.md) guide for details on how to update the design tokens.
 
-See [ember-flight-icons/CONTRIBUTING](packages/ember-flight-icons/CONTRIBUTING.md) guide for details on how to build and test the Ember addon.
+### ember install
 
-## @hashicorp/design-system-tokens
+Run this command from the monorepo root:
 
-| Package                                                                              | Version                                                                                                                         |
-| ------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| **[@hashicorp/design-system-tokens](./packages/tokens/)** <br />Design tokens    | [![npm version](https://badge.fury.io/js/%40hashicorp%2Fdesign-system-tokens.svg)](https://badge.fury.io/js/%40hashicorp%2Fdesign-system-tokens) |
-
-### Contributing
-
-See [tokens/CONTRIBUTING](packages/tokens/CONTRIBUTING.md) guide for details on how to update the design tokens.
-
-## @hashicorp/design-system-components
-
-| Package                                                                              | Version                                                                                                                         |
-| ------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| **[@hashicorp/design-system-components](./packages/components/)** <br />Design System components    | [![npm version](https://badge.fury.io/js/%40hashicorp%2Fdesign-system-components.svg)](https://badge.fury.io/js/%40hashicorp%2Fdesign-system-components) |
-
-### Contributing
+```bash
+yarn workspace <workspace-package-name> run ember install <package>
+```
 
-See [components/CONTRIBUTING](packages/components/CONTRIBUTING.md) guide for details on how to update the components.
+e.g. `yarn workspace components run ember install ember-a11y-refocus`
```

Comment:
Two thoughts here:
* I think moving this into the first section would make sense as that has the other instructions for working with monorepo whereas the sections right above this are about the sub-packages.
* Because yarn 1 is broken this command half works in that it installs the package, but it also generates a `yarn.lock` in the package folder instead of from the root. So if we want to mention this we should also mention that you'll need to remove that file and manually run `yarn` in the root

Reply:
also, the `workspace` is the name of the package, not the folder. so something like:

`yarn workspace @hashicorp/design-system-components run ember install ember-a11y-refocus`

Reply:
good catch, made those changes, ready for another look: https://github.com/hashicorp/design-system/blob/3cf62fa36361a79ea2e719929645ad1d637603e2/README.md